mediathread/projects/models.py

```diff
@@ -407,6 +408,19 @@ def citations(self):
         note_model = models.get_model('djangosherd', 'SherdNote')
         return note_model.objects.references_in_string(self.body, self.author)
 
+    def update_class_references(self):
```

Comment:
This function is *almost* the same as the one in `mediathread/discussions/utils.py`. Seems like we ought to be able to factor out the common functionality.